tests/test_plugins_apps_av_mcafee.py

```diff
@@ -0,0 +1,16 @@
+from dissect.target import Target
+from dissect.target.plugins.apps.av.mcafee import McAfeeMscLogRecord
+
+from ._utils import absolute_path
+
+
+def test_mcafee_plugin_log():
+    for record in Target.open(absolute_path("data/apps/av/mcafee/mcafee.tar.gz")).mcafee.msc():
```

Comment:
I haven't checked the content of the archive, but would it not be cleaner to only use the database file as test files?

Reply:
I recommend this approach because:

1. You can use the same test file from the CLI with the TAR-loader (for demo purposes or manual testing)
2. This also tests the path detection mechanisms

Reply:
Discussed with team, for now this seems to be a better approach indeed. However an additional benefit of this version was that you could use the same test file for a manual test or demo on the CLI.